src/sentry/static/sentry/app/redux/reducers/availableWorkUnit.js

```diff
@@ -0,0 +1,9 @@
+import {resource} from './shared';
```

Comment:
The name available work unit doesn't make sense. Available work definitions (steps) are the ones with samples queuing to it. As it goes for Clarity, all samples are available to everyone.

Reply:
The samples may be queued elsewhere, but this the list of work units that are ready for work. But the name is not clear, I agree.